### DISCUSSION TITLE
iPXE with network adapter

### DISCUSSION THREAD

**Discussion by fs0ciety404** [ipxe/ipxe#1089] [Q&A]
hi, i need so much help with iPXE. 

i am tryin to configure a bootable usb with iPXE, i need to use a network adapter to launch wds.

how can i do?

**Comment by NiKiZe** [comment]
Stackoverflow has some good guides on how to write questions which gives you good answers.

The posted message as it is now seems to ba asking for someone to write a guide for you. That is not so likely to get good responses.
